### ISSUE TITLE
Add a means to check an attendee in to an event or session instance

### ISSUE THREAD

**Issue by pgib** [attendease/attendease-js#3]
If an event allows _self check-ins_, we needs some methods that we can call to check a logged in Attendee into an event or session instance.

Requires https://github.com/coverall/Attendease/issues/5414

**Comment by pgib** [comment]
Something like:

```
Attendease.checkin() // checks into the event
Attendease.checkin(instanceId) // checks into an instance
```

**Comment by mikeric** [comment]
Just published attendease 0.5.0 with checkin capabilities. Closing those.

https://www.npmjs.com/package/attendease
